source/install/troubleshooting.rst

```diff
@@ -19,9 +19,40 @@ General Troubleshooting
 -----------------------
 Some of these suggestions can be done directly, and others may need consultation from your network administrator.
 
-- Take a look at the logs (``mattermost.log`` and NGINX logs) for errors.
-- You can also search the error messages online - existing solutions can often be applied.
-- Open **System Console > General > Logging** and set File Log Level to **DEBUG**. Make sure to revert to **INFO** after troubleshooting to save disk space.
+Take a Look at the Logs
```

Comment:
```suggestion
Review Mattermost Logs
```

Reply:
Propose being more explicit that these are Mattermost-service related logs per the section that follows (as opposed to database logs or proxy logs).

Also, propose using `review` instead of `take a look`, though I'm only 1/5 and is a subjective preference.